files/default/neutron-ha-tool.py

```diff
@@ -171,22 +174,25 @@ def run(args):
     if not auth_version:
         auth_version = os.getenv('OS_IDENTITY_API_VERSION', None)
         if not auth_version:
-            auth_version = 3
+            auth_version = '2.0'
```

Comment:
We add v3 from the commit, but we fallback to v2 if the version is not specified.